.gitignore

```diff
@@ -0,0 +1 @@
+/target/
```

Comment:
do not include this file.  Manually determine what should and should not be included in your commit